### ISSUE TITLE
Warm start not working

### ISSUE THREAD

**Issue by samueltober** [dreamquark-ai/tabnet#373]
**warm_start is not recognized as an input parameter**

**When calling the fit function on the TabNetRegressor and TabNetPretrainer, warm_start is not recognized as a valid input parameter and a TypeError is raised**

**Expected behavior**
Since I can see warm_start is an input parameter to the class, I would expect it to be recognized. 

**Other relevant information:**
python version: 3.6
Operating System: MacOS

**Comment by Optimox** [comment]
Hello @samueltober,

What version of pytorch-tabnet are you using? `warm_start` is still on the development branch at the moment (we should make a release soon)

**Comment by samueltober** [comment]
Hey, thanks for such a quick reply! 

I see, I thought I was on the main branch when I checked. But then i can assume the default is that if fit is ran twice, the previous weights are re-used and trained further?

**Comment by Optimox** [comment]
yes exactly, at the moment the warm_start is not available and automatically set to `True`.

It's not a pig issue, it's just not scikit-compatible so it will be changed in the future.
